core-bundle/src/Mailer/ContaoMailer.php

```diff
@@ -77,10 +78,14 @@ private function setTransport(Message $message): void
     }
 
     /**
-     * Overrides the from address according to the transport.
+     * Overrides the from address according to config.
      */
     private function setFrom(Email $message): void
     {
+        if (null !== $this->overrideFrom) {
+            $this->doSetFrom($message, $this->overrideFrom);
+        }
```

Comment:
The info says _Overrides the "From" address for any emails sent by the mailer, if not otherwise specified by a transport_ but the current implementation does not check the transport if `$this->overrideFrom` is set. Are you sure that this is implemented correctly?